src/SubtitleEdit.csproj

```diff
@@ -1355,6 +1355,9 @@
   </ItemGroup>
   <ItemGroup>
     <None Include="Resources\Transparent Background.png" />
+    <Content Include="..\LICENSE.txt">
+      <Link>LICENSE.txt</Link>

```

Comment:
IIRC the `<Link>` item metadata is used when the file is located outside the project directory. But it is not really necessary here. I would suggest item `<Content Include="..\LICENSE.txt" />` without metadata (i.e., replace `..\gpl.txt` with `..\LICENSE.txt`).